docs/development/workflow/get_devel_version.rst

```diff
@@ -222,30 +222,43 @@ of `Astropy`_, for now this straightforward way will work (if you want to jump
 ahead to the more sophisticated method look at :ref:`virtual_envs`).
 
 .. note::
-    There are a couple of circumstances in which this quick method of
+    There are some circumstances in which this quick method of
```

Comment:
Since there is now only one circumstance left, can we just write:
```
If you want to work on C or Cython code in `Astropy`, this quick method
of activating your copy of `Astropy`_ will *not* work _ -- you need to go 
straight to using a virtual python environment.
```